Makefile

```diff
@@ -200,110 +192,54 @@ realclean: clean
 shell:
 	$(SHELL) -i
 
-images: image-controller-manager image-flex-volume-driver image-provisioner image-csi-plugin image-k8s-keystone-auth image-octavia-ingress-controller image-manila-provisioner image-manila-csi-plugin image-kms-plugin image-magnum-auto-healer
-
-image-controller-manager: work openstack-cloud-controller-manager
```

Comment:
Can we keep `image-controller-manager` as it's also used in https://github.com/theopenlab/openlab-zuul-jobs/blob/9ac8bdf0e56c6f64a88224bab3956460f4bd8b8d/playbooks/cloud-provider-openstack-acceptance-test-lb-octavia/run.yaml#L60

Reply:
sure, we should change [1] to use `image-openstack-cloud-controller-manager` directly

[1] https://github.com/theopenlab/openlab-zuul-jobs/blob/9ac8bdf0e56c6f64a88224bab3956460f4bd8b8d/playbooks/cloud-provider-openstack-acceptance-test-lb-octavia/run.yaml#L60